main_dnn.py

```diff
@@ -190,23 +190,23 @@ def train(args):
         loss.backward()
         optimizer.step()
         
-        if iter % 200 == 0:
-            print("Iter: %d loss: %f" % (iter, loss))
+        if iter % 50 == 0:
```

Comment:
Why?

Reply:
I committed a file that I modified incorrectly.

Reply:
I committed incorrect file.